packages/design-system/src/components/Tag/Tag.cy.tsx

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable testing-library/await-async-query */
+/* eslint-disable testing-library/prefer-screen-queries */
+import React from 'react';
+
+import { Tag } from '.';
+
+context('<Tag />', () => {
+	it('should render', () => {
+		cy.mount(<Tag data-testid="my.tag" />);
```

Comment:
Note for later: it could be good to test all variants and props OR put test directly on variant files

Reply:
I have added some but because it is css module and the variant only apply a class, I am not able to check the class (it is hashed)